Source/info.c

```diff
@@ -185,6 +185,37 @@ void A_Detonate();        // killough 8/9/98: detonate a bomb or other device
 void A_Mushroom();        // killough 10/98: mushroom effect
 void A_BetaSkullAttack(); // killough 10/98: beta lost souls attacked different
 
+// [XA] New mbf21 codepointers
```

Comment:
I don't think these are necessary here, because no element in `states[]` is actually initialized with one of these pointers.